test/github_incoming_test.go

```diff
@@ -47,10 +69,10 @@ func TestGithubAppIncomingForDifferentEvent(t *testing.T) {
 	}, randomedString, randomedString, triggertype.PullRequest.String(), map[string]string{})
 	assert.NilError(t, err)
 
-	verifyIncomingWebhook(t, randomedString, entries)
+	verifyIncomingWebhook(t, randomedString, entries, false, false)
 }
 
-func verifyIncomingWebhook(t *testing.T, randomedString string, entries map[string]string) {
+func verifyIncomingWebhook(t *testing.T, randomedString string, entries map[string]string, onWebhook, onSecondController bool) {
 	ctx := context.Background()
 	ctx, runcnx, opts, ghprovider, err := tgithub.Setup(ctx, false, false)
```

Comment:
I think we need to pass down `onWebhook, onSecondController` values to 
ctx, runcnx, opts, ghprovider, err := tgithub.Setup(ctx, onSecondController, onWebhook)  :thinking: 
otherwise its always GitHubApp and primary controller isn't it ?

Reply:
well spotted, my test was actually not working...

Reply:
this is harder to fix, since involve infra running change

Reply:
ah i see

Reply:
it was fixed fyi 🙃